### ISSUE TITLE
Unable to install other modules after installing this

### ISSUE THREAD

**Issue by kevupton** [ReactiveX/rxjs-tslint#59]
Cannot install modules after install this package,
Have to uninstall first

```
error An unexpected error occurred: "could not find a copy of typescript to link in ...\\node_modules\\rxjs-tslint\\node_modules".
```

Typescript: 3.1.6
Rxjs-Tslint: 0.1.6

**Comment by kuiperr005** [comment]
Same issue here

**Comment by narkaTee** [comment]
Do you use yarn?

I've trace the problem down to c30cd0c5945787ad6e94312c2403ae65e58fa50b which added the typescript dependency and yarn seems to have issues linking the 2.x typescript version if your project uses 3.x

**Comment by kuiperr005** [comment]
I'm using yarn and having as devDependency typescript version 2.7.x

**Comment by narkaTee** [comment]
> I'm using yarn and having as devDependency typescript version 2.7.x

Try upgrading typescript to something matching `^2.8.3` **or** downgrade rxjs-tslint to 0.1.5

**Comment by vsviridov** [comment]
Why does this need to put `typescript` as a runtime dependency? Compiled rules should just load whatever definitions for `ts.SyntaxKind` the project is using...

**Comment by jdforsythe** [comment]
This has basically broken `yarn install` for anyone using Angular v7 (typescript ~3.1.6) with rxjs-tslint. I created a PR to update the dependency definition.

**Comment by Necroskillz** [comment]
I'm still getting this error with `0.1.7` (typescript `~3.2.0`)

**Comment by XuXiaowen** [comment]
> 安装此软件包后无法安装模块，
> 必须先卸载
> 
> ```
> error An unexpected error occurred: "could not find a copy of typescript to link in ...\\node_modules\\rxjs-tslint\\node_modules".
> ```
> 打字稿：3.1.6 
> Rxjs-Tslint：0.1.6

Right!!

**Comment by zhongsp** [comment]
The error still exists with  "rxjs-tslint": "^0.1.7" and `"typescript": "~3.5.3"`.
Making `rxjs-tslint` fixed to `0.1.5` works.

**Comment by jblayneyXpanxion** [comment]
Still having this issue with typescript 4.1.6 and rxjs-tslint 0.1.8 AND 0.1.5

**Comment by JounQin** [comment]
See https://github.com/ReactiveX/rxjs-tslint/pull/68#issuecomment-580797554

You can replace `rxjs-tslint` with `@rxts/rxjs-tslint`.
